infrastructure/docker/celo/start.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+mkdir -p /data/keystore
+cp init/keystore/* /data/keystore
+cp init/password.txt /data/password.txt
+[ ! -d "/data/celo" ] && geth --nousb --datadir=/data init init/genesis.json
+
+ geth --port 3000 --networkid 1101 --identity "somerandomidentity" \
+    --ws --wsaddr "0.0.0.0" --wsport "8546" --wsorigins "*" \
+    --rpc --rpcport "8545" --rpcaddr "0.0.0.0" --rpccorsdomain "" \
```

Comment:
Thoughts on using different default ports for RPC and WS, not to cause confusion with Ethereum ones?

Reply:
We use the same local network in truffle config for Celo and Ethereum (see my previous comment). I also do not think right now we'll be using those two chains at the same time. Maybe in the future but then, we'll have to make it parametrizable and think about what to do with truffle config.